src/ggrc/models/relationship.py

```
@@ -62,7 +62,7 @@ def destination_attr(self):
 
   @property
   def destination(self):
-    return getattr(self, self.destination_attr)
+    return getattr(self, self.destination_attr, None)
```

Hey vishal, thanks for the PR.

My worry is that if the code is failing at this point it's a sign that something is seriously wrong. I think we shouldn't just silently fail if this happens. 

Can you investigate what is the root cause of this issue?